phpcs.xml.dist

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
 <ruleset name="Roave/BetterReflection coding standard">
+    <config name="php_version" value="70400"/>
```

Comment:
We should have this at `80000` - if this is not to be done here, we need to move it to a separate issue to be put under the same milestone

Reply:
I have already a draft: https://github.com/Roave/BetterReflection/pull/753